packages/cli/src/ceramic-daemon.ts

```diff
@@ -738,6 +741,32 @@ export class CeramicDaemon {
     }
   }
 
+  async validateAdminRequest(req: Request, res: Response, next:NextFunction): Promise<void> {
+    if (!req.headers.authorization) {
+      res.status(StatusCodes.UNAUTHORIZED).json({ error: 'Missing authorization header' })
+      return
+    }
+    const jwsString = req.headers.authorization.split('Basic ')[1]
```

Comment:
is this a little brittle?  maybe use https://www.npmjs.com/package/basic-auth ?

Reply:
probably, less familiar with the codebase here, so mostly tried to align as closely as possible with existing admin api and minimize changes in any other places (some parts are somewhat redundant now tho)

Reply:
@zachferland , can you add a ticket to unify the code behind `admin/models` with what you wrote in this PR? I think that we'll have some (almost) duplicated code when we merge this PR.